symplyphysics/core/fields/operators.py

```diff
@@ -2,10 +2,46 @@
 
 from ..dimensions import ScalarValue
 from ..fields.vector_field import VectorField
+from ..fields.scalar_field import ScalarField
 from ..coordinate_systems.coordinate_systems import CoordinateSystem
 from ..vectors.vectors import Vector
 
 
+def gradient_operator(field: ScalarField) -> VectorField:
```

Comment:
It is more usual to have Vector as a result of gradient operation.

Reply:
Isn't it better to have `VectorField` in case the end user has to plug the result of `gradient_operator` into other functions that accept `VectorField` instead of `Vector`, like `divergence_operator`? And as far as I understand, to get a `Vector` from `VectorField` it's as easy as to call `.apply_to_basis()` on the latter.

Reply:
Sometimes it results with plain scalar values and it does not make sense to make a vector field from them. I believe making it return Vector would make code cleaner.

Reply:
I agree with you now